### PR TITLE
modified the README file according to the feedback.

### PR DESCRIPTION
**Pull request by jahid1209**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,6 @@
 
 This repository contains a Jenkins plugin implemented as a Maven project. The plugin provides functionality for performing Synopsys Security Scan with Black Duck, Coverity and Polaris. This README.md file serves as a guide for the users of the plugin. Please note that this Jenkins plugin currently supports only **Bitbucket as the source code management (SCM) system.**
 
-## Build
-
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-
```
Comment (zaman-akib) [inline README.md:8]: This shouldn't be removed

**Review by zaman-akib — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dmamidisynopsys) [reply README.md]: @jahid1209 Can you elaborate on which feedback? Was this from Infra team?

**Review by dmamidisynopsys — COMMENTED** [review]
Added in general section

File: README.md (hunk 1 above)
Comment (jahid1209) [reply README.md]: @dmamidisynopsys 
I meant that Tithi made some suggestion regarding sample pipeline in the README.md. Based on those suggestions **Maksud** made some changes. But he doesn't have the write permission to create a PR. Therefore, I created a PR on his behalf.

**Review by dmamidisynopsys — APPROVED** [review]

**Pull request merged**